src/OpaEarning/Build/Deploy/1718/sql/ddl/OPA.Transient.Rulebase.Ilr.ddl.tables.sql

```diff
@@ -0,0 +1,1537 @@
+--Rulebase schema
+
+if not exists(select schema_id from sys.schemas where name='Rulebase')
+begin
+	exec('create schema Rulebase')
+end
+GO
+
+if object_id('[Rulebase].[EFA_SFA_Cases]','u') is not null
+begin
+	drop table [Rulebase].[EFA_SFA_Cases]
+end
+GO
+
+create table [Rulebase].[EFA_SFA_Cases]
+(
+	[LearnRefNumber] varchar(12),
```

Comment:
The tables created in this file have LearnRefNumber as varchar(12), but elsewhere in this pull request (e.g. OPA.Transient.Invalid.ddl.tables.sql and OPA.Transient.Input.ddl.tables.sql), LearnRefNumber is changed from varchar(12) to varchar(100). Is this by design?